### PR TITLE
[DW-128] DW-129 Add news hub page

### PR DESCRIPTION
**Pull request by robValtech**

### CODE REVIEW

File: repository-data/application/src/main/resources/hcm-config/hst/configurations/common/pages/static-ui-news-hub.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (giacomolm) [inline repository-data/application/src/main/resources/hcm-config/hst/configurations/common/pages/static-ui-news-hub.yaml]: I still need to understand why there are static ui configuration, but this is not relevant for this review

File: repository-data/webfiles/src/main/resources/site/freemarker/common/static-ui/news-hub-page.ftl [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="grid-wrapper grid-wrapper--article">
+        <div class="grid-row">
+            <div class="column column--one-third page-block page-block--sidebar sticky sticky--top">
+                <#assign filterNavLinks = [{ "url": "#", "title": "NHS Digital (4)" }, { "url": "#", "title": "Programme (16)" }, { "url": "#", "title": "NHS Digital (32)" }, { "url": "#", "title": "System and services (22)" }, { "url": "#", "title": "2018 (25)" }, { "url": "#", "title": "2017 (150)" }, { "url": "#", "title": "2016 (123)" }, { "url": "#", "title": "2015 (85)" }]/>
```
Comment (ngta1) [inline repository-data/webfiles/src/main/resources/site/freemarker/common/static-ui/news-hub-page.ftl]: Do the numbers in brackets denote the number of say 2017 news articles?  Are they intentionally hardcoded or were they a placeholder to be replaced with dynamic value?

Comment (robValtech) [reply repository-data/webfiles/src/main/resources/site/freemarker/common/static-ui/news-hub-page.ftl]: @ngta1 Everything under `static-ui` is for getting rapid UI work done without any backend - it also gives us a glimpse of the UI patterns and components we're using without the need of any dynamic data.

Comment (ngta1) [reply repository-data/webfiles/src/main/resources/site/freemarker/common/static-ui/news-hub-page.ftl]: Ah, of course.  Overlooked the static-ui part.  Thanks for clarifying.

**Pull request merged**